src/components/DropdownV2/DropdownV2-test.js

```diff
@@ -57,4 +57,33 @@ describe('DropdownV2', () => {
       selectedItem: mockProps.items[1],
     });
   });
+
+  describe('should display initially selected item found in `initialSelectedItem`', () => {
+    it('using an object type for the `initialSelectedItem` prop', () => {
+      const wrapper = mount(
+        <DropdownV2 {...mockProps} initialSelectedItem={mockProps.items[0]} />
+      );
+
+      expect(
```

Comment:
As a code formatting comment, could we do:

```js
expect(wrapper.find('span.bx--list-box__label').text()).toEqual(mockProps.items[0].label);
```

Reply:
Good point.

Reply:
Done ✔️